BunqSdk/Context/ApiEnvironmentType.cs

```diff
@@ -10,33 +10,33 @@ public sealed class ApiEnvironmentType
         /// <summary>
         /// Mapping of environment type to the base URI.
         /// </summary>
-        private static readonly IDictionary<string, string> NAME_TO_BASE_URI_MAP = new Dictionary<string, string>
+        private static readonly IDictionary<string, string> NameToBaseUriMap = new Dictionary<string, string>
         {
-            {ENVIRONMENT_TYPE_PRODUCTION, BASE_URI_PRODUCTION},
-            {ENVIRONMENT_TYPE_SANDBOX, BASE_URI_SANDBOX}
+            {EnvironmentTypeProduction, BaseUriProduction},
+            {EnvironmentTypeSandbox, BaseUriSandbox}
         };
 
         /// <summary>
         /// Production environment constants.
         /// </summary>
-        public static readonly ApiEnvironmentType PRODUCTION = new ApiEnvironmentType(ENVIRONMENT_TYPE_PRODUCTION);
-        private const string ENVIRONMENT_TYPE_PRODUCTION = "PRODUCTION";
-        private const string BASE_URI_PRODUCTION = "https://api.bunq.com/v1/";
+        public static readonly ApiEnvironmentType Production = new ApiEnvironmentType(EnvironmentTypeProduction);
+        private const string EnvironmentTypeProduction = "PRODUCTION";
+        private const string BaseUriProduction = "https://api.bunq.com/v1/";
 
         /// <summary>
         /// Sandbox environment constants.
         /// </summary>
-        public static readonly ApiEnvironmentType SANDBOX = new ApiEnvironmentType(ENVIRONMENT_TYPE_SANDBOX);
-        private const string ENVIRONMENT_TYPE_SANDBOX = "SANDBOX";
-        private const string BASE_URI_SANDBOX = "https://sandbox.public.api.bunq.com/v1/";
+        public static readonly ApiEnvironmentType Sandbox = new ApiEnvironmentType(EnvironmentTypeSandbox);
```

Comment:
We might not want to refactor `SANDBOX` as this is normally how we name environment constants.

Reply:
Invalid due to https://github.com/bunq/sdk_csharp/pull/71#discussion_r158918468